cmk/base/legacy_checks/raritan_pdu_plugs.py

```diff
@@ -33,12 +33,14 @@ def check_raritan_pdu_plugs(item, params, parsed):
     if data.get("outlet_name"):
         yield 0, data["outlet_name"]
 
+    required_state = params.get("required_state", params["discovered_state"])
+    discovered_state = params["discovered_state"]
     state, state_info = data["state"]
-    yield state, "Status: %s" % state_info
 
-    required_state = params.get("required_state", params["discovered_state"])
     if state_info != required_state:
-        yield 2, "Expected: %s" % required_state
```

Comment:
@mo-ki should we include the "CRIT -" and "OK -" in the summary? This should be picked up by the respective state code right and therefore redundant?

Reply:
Yes. Same goes for the `(!!)`.